.github/workflows/ruby.yml

```diff
@@ -6,7 +6,7 @@
 # For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
 
 name: CI
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
will the CI run if I push more commits to the already opened PR?

Reply:
Yeah, `pull_request` actually means every action done on a pull request. Well not every action but many of them

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
ah, great then